liveblog.php

```diff
@@ -454,7 +454,7 @@ public static function is_liveblog_post( $post_id = null ) {
 	 * @return bool
 	 */
 	public static function is_viewing_liveblog_post() {
-		return (bool) ( is_single() && self::is_liveblog_post() );
+		return (bool) ( ( is_single() || is_page() || is_attachment() ) && self::is_liveblog_post() );
```

Comment:
Does this take into account the custom post types that can be added through the filter?